### PR TITLE
LPS-72365

### PR DESCRIPTION
**Pull request by Ithildir**
Hi,
Please let this pull finish.

Thank you!

cc @gamerson  and @lawrence-lee

### CODE REVIEW

**Comment by Ithildir** [comment]
ci:reopen

File: modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java [hunk 1]
```diff
@@ -150,6 +159,34 @@ private static int _updateStartedAppServerStopCounters(
...
+			"Configures additional OSGi modules to deploy during integration " +
+				"testing.");
+		configuration.setVisible(false);
+
+		return configuration;
+	}
+
+	private void _addDependenciesTestModules(Project project) {
+		GradleUtil.addDependency(
+			project, TEST_MODULES_CONFIGURATION_NAME, "org.apache.aries.jmx",
```
Comment (gamerson) [inline modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java]: Do we not need jmx.api and org.apache.aires.util?  are those included in transitive dependencies?

File: modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java [hunk 2]
```diff
@@ -150,6 +159,34 @@ private static int _updateStartedAppServerStopCounters(
...
+	private Configuration _addConfigurationTestModules(final Project project) {
+		Configuration configuration = GradleUtil.addConfiguration(
+			project, TEST_MODULES_CONFIGURATION_NAME);
+
+		configuration.defaultDependencies(
+			new Action<DependencySet>() {
+
+				@Override
+				public void execute(DependencySet dependencySet) {
+					_addDependenciesTestModules(project);
```
Comment (gamerson) [inline modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java]: Do we always add?  I didn't see where we check to see if aires is already there.

**Comment by Ithildir** [comment]
⚠️ @brianchandotcom, please **DON'T** merge this pull, it's causing test failures! ⚠️

File: modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java (hunk 1 above)
Comment (Ithildir) [inline modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java]: Yeah, they're transitive dependencies. The `testModules` configuration is transitive by default.

File: modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java (hunk 2 above)
Comment (Ithildir) [inline modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java]: The code is [here](https://github.com/brianchandotcom/liferay-portal/pull/48816/files#diff-0552859495dfc80071edb3a7cc8865b6R547). I add both the configuration and the `osgi/test` dir as source of that `copy` command, telling Gradle to exclude duplicates.
